Source/UI/PaymentForm/View/SecurityCodeViewModel.swift

```diff
@@ -42,8 +42,8 @@ final class SecurityCodeViewModel {
         }
 
         guard let cleanedInput = newInput?.decimalDigits,
-              !cleanedInput.isEmpty,
-              cleanedInput.count <= inputMaxLength else {
+              cleanedInput.count <= inputMaxLength ||
+              cleanedInput.isEmpty else {
```

Comment:
We could write an unit test for this? Was the issue that removing input was not possible?

Reply:
Do we actually need to check if input is empty at all?